tck/src/main/java/ee/jakarta/tck/data/standalone/entity/EntityTests.java

```diff
@@ -393,8 +336,6 @@ public void testBasicRepositoryMethods() {
         assertEquals(104, box5.width);
         assertEquals(185, box5.height);
 
-        // BasicRepository.deleteByIdIn
-        boxes.deleteByIdIn(List.of("TestBasicRepositoryMethods-05"));
 
         TestPropertyUtility.waitForEventualConsistency();
```

Comment:
The test logic that check for the result to drop to 0 is wrong if you remove deletion. The following suggestion adds deletion back in, but using deleteById which we still have in the spec,
```suggestion
        // BasicRepository.deleteById
        boxes.deleteById("TestBasicRepositoryMethods-05");

        TestPropertyUtility.waitForEventualConsistency();
```